lib/internal/cluster/round_robin_handle.js

```diff
@@ -1,17 +1,58 @@
 'use strict';
 const assert = require('assert');
 const net = require('net');
+const Scheduler = require('internal/cluster/scheduler');
 const { sendHelper } = require('internal/cluster/utils');
-const getOwnPropertyNames = Object.getOwnPropertyNames;
+const { create, getOwnPropertyNames } = Object;
```

Comment:
The variable name aliasing for this style is even harder to remember/get used to.